ql/src/java/org/apache/hadoop/hive/ql/ddl/process/show/compactions/ShowCompactionsDesc.java

```diff
@@ -32,7 +32,8 @@
   private static final long serialVersionUID = 1L;
 
   public static final String SCHEMA =
-      "compactionid,dbname,tabname,partname,type,state,hostname,workerid,enqueuetime,starttime,duration,hadoopjobid,errormessage#" +
+      "compactionid,dbname,tabname,partname,type,state,hostname,workerid,enqueuetime,starttime,duration,hadoopjobid," +
+      "errormessage#" +
```

Comment:
Remove this formatting change.  Little value and adds to this already large review.

Reply:
There is a 120 limit in the Hive checkstyle, and I'm trying to make all DDL codes checkstyle violation free. This patch is about making Show kind commands cleaner, that is why it is here.

Reply:
Added @formatter:off - @formatter:on instead